pkg/minikube/cluster/cluster.go

```diff
@@ -371,16 +373,32 @@ func GetHostDockerEnv(api libmachine.API) (map[string]string, error) {
 }
 
 // GetHostLogs gets the openshift logs of the host VM.
-func GetHostLogs(api libmachine.API) (string, error) {
+// If follow is specified, it will tail the logs
+func GetHostLogs(api libmachine.API, follow bool) (string, error) {
```

Comment:
Yes, it is inherited from Minikube fork, I was wondering the same. will update the name and also move it to openshift domain